staging/src/k8s.io/cloud-provider/app/builder.go

```diff
@@ -59,6 +64,10 @@ func (cb *CommandBuilder) AddFlags(additionalFlags cliflag.NamedFlagSets) {
 	cb.additionalFlags = additionalFlags
 }
 
+func (cb *CommandBuilder) WithFeatureGates(featureGates featuregate.MutableFeatureGate) {
```

Comment:
@elmiko I think this fulfills the original intent: the default is the same as it always was, but now it is possible for a caller to invoke this with a custom set of featuregates that includes the stock featuregates and their custom additions..

Reply:
i looked through several popular cloud-provider (gcp, aws, azure, ibm, nutanix, vsphere) and i'm not seeing usage of this CommandBuilder. perhaps folks are using it another manner, but i will share these changes at our next sig meeting to see if there are any wider comments.